README.md

```diff
@@ -251,6 +251,7 @@ The supported rules are described below:
 | no_property_description     | Flag any schema that contains a 'property' without a `description` field.     | shared   |
 | description_mentions_json   | Flag any schema with a 'property' description that mentions the word 'JSON'.  | shared   |
 | array_of_arrays             | Flag any schema with a 'property' of type `array` with items of type `array`. | shared   |
+| compose_model_field_not_array | Flag any compose model schema that does not use an array.                   | shared   |
```

Comment:
If we keep this configurable, I think we want a clearer name and description.  "compose model" is a term from the SDK generator -- not used in OpenAPI or JSON Schema.  So better to use something like "allOf_value_not_array" and "Flag any `allOf` schema attribute whose value is not an array".

Reply:
I chose to use compose model because this PR checks that `allOf`, `anyOf, and `oneOf` use arrays. In my next changes, I am going to remove the configuration option, but I want to make sure you agree with the choice to include `anyOf` and `oneOf`.

Reply:
Yes -- we should check all three.  I made my suggestion before I saw the code below.